cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueAdmin.java

```diff
@@ -418,16 +414,9 @@ private void createConfigTable() throws IOException {
   }
 
   @Override
-  public void dropAll() throws Exception {
-    dropTablesWithPrefix(tableNamePrefix);
-    // drop config table last
-    drop(Bytes.toBytes(configTableName));
-  }
-
-  @Override
   public void dropAllInNamespace(String namespaceId) throws Exception {
     // Note: The trailing "." is crucial, since otherwise nsId could match nsId1, nsIdx etc
-    dropTablesWithPrefix(String.format("%s.%s.", tableNamePrefix, namespaceId));
+    dropTablesWithPrefix(String.format("%s.", getTableNamePrefix(namespaceId)));
```

Comment:
Right. the changes i made here still allow dropallInNamespace to work, but it'll definitely be different when we add hbase namespaces. Let's discuss this tomorrow?